README.md

```diff
@@ -183,6 +183,14 @@ Note: if an endpoint is not listed here, a complete list can be retrieved by run
 
 
 - `GET /api/v1/certificates`: Fetch all certificates
```

Comment:
I think it should be important to specify the response the url gives when not adding the optional Json, front-end team needs all these explanations with pebbles.

Reply:
done, i've now documented that in Certificates section